libs/components/src/spinner/spinner.component.ts

```diff
@@ -0,0 +1,66 @@
+import { coerceBooleanProperty } from "@angular/cdk/coercion";
+import { CommonModule } from "@angular/common";
+import { Component, HostBinding, Input } from "@angular/core";
+
+import { I18nService } from "@bitwarden/common/platform/abstractions/i18n.service";
+
+@Component({
+  selector: "bit-spinner",
+  templateUrl: "spinner.component.html",
+  standalone: true,
+  imports: [CommonModule],
+})
+export class SpinnerComponent {
+  /**
+   * The size of the spinner. Defaults to `large`.
+   */
+  @Input() size: "fill" | "small" | "large" = "large";
+
+  private _noColor = false;
+  /**
+   * Disable the default color of the spinner, inherits the text color.
+   */
+  @Input()
+  get noColor(): boolean {
+    return this._noColor;
+  }
```

Comment:
I think you can do it like this and then you don't need the internal property? Same for `sr`
```suggestion
  @Input({ transform: booleanAttribute })
  noColor = false;
```

Reply:
I was certain that wasn't supported in the version we use. Looks like I was wrong nice,